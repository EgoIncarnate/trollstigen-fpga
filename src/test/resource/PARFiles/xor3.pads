#block name	x	y	subblk	block number
#----------	--	--	------	------------
top^b_in	5	0	0	#2
top^a_in	6	0	0	#3
top^c_in	9	1	0	#4
out:top^z_out	7	0	0	#5
top^clock	1	0	0	#8
