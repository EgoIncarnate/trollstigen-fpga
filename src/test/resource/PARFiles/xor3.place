Netlist file: xor3.net   Architecture file: troll_arch8.xml
Array size: 8 x 16 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
top^BITWISE_XOR~1^LOGICAL_XOR~3	7	1	0	#0
top^BITWISE_XOR~0^LOGICAL_XOR~4	6	1	0	#1
top^b_in	5	0	-1	#2
top^a_in	6	0	-1	#3
top^c_in	9	1	-1	#4
out:top^z_out	7	0	-1	#5
vcc		3	16	0	#6
gnd		1	13	0	#7
top^clock	1	0	-1	#8
