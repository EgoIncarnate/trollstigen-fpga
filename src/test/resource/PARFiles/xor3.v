// Test enable circuitry

module  xor3(clock,
                a_in,
                b_in,
                c_in,
                z_out
                );

input   clock;
input   a_in;
input b_in;
input c_in;
reg temp;
output z_out;

always @(posedge clock)
begin
  temp <= a_in ^ b_in ^ c_in;
end

assign z_out = temp;

endmodule

