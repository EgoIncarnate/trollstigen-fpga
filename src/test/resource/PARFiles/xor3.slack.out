The following slacks have been normalized to be non-negative by relaxing the required times to the maximum arrival time.

Largest slack in design: --
Smallest slack in design: --
Total slack in design: 0
Total negative slack: 0


Net #	Driver_tnode	to_node	Slack

    0	   30		    0	--
    1	   17		    1	--
    2	    2		   33	--
    3	   36		    3	--
    4	   24		   15	--
    5	   27		   16	--
